app/controllers/users_controller.rb

```diff
@@ -1,10 +1,12 @@
 class UsersController < ApplicationController
+  before_action :authenticate_user!
   def index
     @users = User.all
   end
 
   def show
-    @user = User.find(params[:id])
+    # @user = User.find(params[:id])
```

Comment:
- [ ] Kindly remove any unnecessary code for a cleaner appearance and better optimization. 👍